### DISCUSSION TITLE
Allow custom URL when inserting image

### DISCUSSION THREAD

**Discussion by iandrianto** [directus/directus#16265] [Feature Requests]
Directus already have the features for uploading files to a cloud storage.

The problem when we insert an image in the WYSIWYG editor, the inserted URL will point to Directus local. (e.g. http://0.0.0.0:8055/assets/...). That's not what we want. The URL should be able to set to a custom URL (e.g. GCS or CDN URL) because most of the time we won't access the files from Directus server.

Is that feature already available? If not, it would be better to have a feature for setting a custom default base image URL.

**Comment by azrikahar** [comment]
It's not available yet, but there is discussion on this topic in #7612 👍

In the mean time, although this isn't ideal, you should be able to click "Edit Source Code" and change the URL manually.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
